docker/package/scripts/octez-node

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
```

Comment:
A tiny nitpick: the name of this script is the same as binary, which can be confusing (at least it confused me), AFAICS, it's renamed during the actual installation, but I still propose to rename it to something like `octez-node-network`